### PR TITLE
Add various nodePackages

### PR DESCRIPTION
**Pull request by winston0410**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by winston0410** [comment]
@SuperSandro2000 A friendly ping, and I have merged the commits from upstream to latest.

File: pkgs/development/node-packages/node-packages.json [hunk 1]
```diff
@@ -1,10 +1,15 @@
 [
   "@angular/cli"
+, "@angular/language-server"
 , "@antora/cli"
 , "@antora/site-generator-default"
 , "@bitwarden/cli"
+, "@elm-tooling/elm-language-server"
```
Comment (SuperSandro2000) [inline pkgs/development/node-packages/node-packages.json:7]: dupe with https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/elm/packages/node-packages.json

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/node-packages/node-packages.json (hunk 1 above)
Comment (winston0410) [reply pkgs/development/node-packages/node-packages.json]: Just removed that, and added rustywind and merged with upstream.

**Comment by winston0410** [comment]
Closing this as no response.

**Pull request closed without merging**